### DISCUSSION TITLE
Bruno logo

### DISCUSSION THREAD

**Discussion by pove** [usebruno/bruno#464] [Polls]
@helloanoop, I hope I don't offend your buddy... I am not a designer.

### Current
![image](https://github.com/usebruno/bruno/assets/10817653/660e393e-fff5-492d-bcee-dedc0c2e7316)

### My first test
![image](https://github.com/usebruno/bruno/assets/10817653/863136df-b3de-4f62-bced-877188a89b59)

### I want something very simple
![image](https://github.com/usebruno/bruno/assets/10817653/a164eaca-7de4-4916-a6bb-6b980910188c)

### Ask AI for help
![image](https://github.com/usebruno/bruno/assets/10817653/b5984ba6-a58c-46bb-8fef-65b2e7f1fef1)

### Same in negative
![image](https://github.com/usebruno/bruno/assets/10817653/440326e4-2869-4ea9-9976-14654612f002)

### Think in positive
![image](https://github.com/usebruno/bruno/assets/10817653/edfe6f6f-ec32-4c39-ab0d-c5d2d30e92d9)

### Current modified
![image](https://github.com/usebruno/bruno/assets/10817653/fb207f18-3c32-4772-9669-44e3a2b55253)

**Comment by trin4ik** [comment]
AI one love )

**Comment by helloanoop** [comment]
Bruno's logo is sourced from https://openmoji.org/library/emoji-1F436 ❤️
I have added attribution in the website. Need to add it in the app too.

Here are some other beautiful logo's from the author: [Sofie Ascherl](https://openmoji.org/library/#author=Sofie%20Ascherl)

**Comment by pove** [comment]
Not looking so good in Windows left-corner icon:

![image](https://github.com/usebruno/bruno/assets/10817653/5f64b607-048c-495f-ba21-ae637f5ffcb3)

**Comment by x0vr** [comment]
I agree that this topic is maybe worth to be discussed.
In my opinion, the style of the image (of this author) is not well suited for a logo of an app. (no front)

**Comment by helloanoop** [comment]
@pove I am actually starting to like the logo. Which AI did you use 😊?

**Reply by pove** [reply]
DALL•E 3 through MS Bing

**Comment by pove** [comment]
If you want the SVG files:
[dog-face-side-negative](https://github.com/usebruno/bruno/assets/10817653/bbdec8d4-7749-4f63-aa71-9a57f413607c)
[dog-face-side](https://github.com/usebruno/bruno/assets/10817653/0053835e-b044-441f-bf8c-4d87e3026dfc)
[dog-face-side-positive](https://github.com/usebruno/bruno/assets/10817653/a9fff07a-121a-4519-bc20-79ea8a04d3bb)

Also added one more conservative...
[dog-face-simliar-circle](https://github.com/usebruno/bruno/assets/10817653/66251950-2ef5-4468-bb54-908eab87f103)

**Comment by koriit-kontakt** [comment]
I say we take a photo of the actual Bruno and turn it into the logo. What do you say? Would make for an interesting story.
